src/react/ui-elements/Container.jsx

```diff
@@ -1,3 +1,4 @@
+import { padding } from '@scality/core-ui/dist/style/theme';
 import styled from 'styled-components';
 
 const MainContainer = styled.div`
```

Comment:
Should we set the default `font-size`, `color`, ... value here instead of in `style.css`'s body? 
Also, I do not see a default/base font `color` set, is it something that will be handled later?

Reply:
done